packages/project-config/src/test/index_test.ts

```diff
@@ -551,23 +539,23 @@ suite('Project Config', () => {
 
   suite('loadOptionsFromFile()', () => {
     test('throws an exception for polymer.json with invalid syntax', () => {
-      const filepath = path.join(__dirname, 'polymer-invalid-syntax.json');
+      const filepath = path.resolve('test/polymer-invalid-syntax.json');
```

Comment:
Can we assume the cwd in tests? I'd been going off of the assumption that file-relative paths with `__dirname` are safer.

Reply:
Seems we can. I've run this test from lerna from the root and local to the package with npm.